src/index.ts

```diff
@@ -32,13 +35,18 @@ const devcardURL = (user_id: string): string =>
 			throw new Error('Filename is required')
 		}
 
+		// throw an error if type is invalid, must be either "default" or "wide"
+		if (type && !['default', 'wide'].includes(type)) {
```

Comment:
Thank you for the effort and sorry for another comment. 

I think a cleaner solution would be to define `DevCardType` as enum, because you can then do
```typescript
Object.values(DevCardType).includes(type)
```

So whenever the enum changes, this check would automatically work as well without the need to change the code.

Reply:
Thank you. I will update soon.

Reply:
Hi @kopancek, I've updated.